tests/integration_tests/validation/conformance_suite_configurations/xbrl_oim_1_0.py

```diff
@@ -7,6 +7,10 @@
         '--httpsRedirectCache',
         '--plugins', 'loadFromOIM',
     ],
+    expected_failure_ids=frozenset([
+        '600-xml/index-xbrl-xml.xml/V-05',
+        '600-xml/index-xbrl-xml.xml/V-06',
```

Comment:
Perhaps open a ticket to remind to clean this up after a new OIM zip is published, and perhaps a comment with a link to that issue?

Reply:
XT-3052